dev/deps.py

```diff
@@ -1,881 +0,0 @@
-# coding: utf-8
```

Comment:
This file should be restored, unless you can articulate how you've replaced all of the functionality.